$NetBSD: patch-CMakeLists.txt,v 1.2 2019/12/30 17:48:05 adam Exp $

Do not make application bundle on Darwin.
Avoid
	CMake Error at CMakeLists.txt:3021 (install):
  	install FILES given directory
  	"${WRKDIR}/.buildlink/cmake-Modules" to install.

--- CMakeLists.txt.orig	2019-12-18 18:27:56.000000000 +0000
+++ CMakeLists.txt
@@ -1359,7 +1359,7 @@ else ()
 	set(EXTCAP_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/extcap" CACHE INTERNAL "The extcap dir")
 endif()
 
-if(APPLE)
+if(APPLE AND ENABLE_APPLICATION_BUNDLE)
 	#
 	# As https://developer.apple.com/library/archive/technotes/tn2206/_index.html
 	# says,
@@ -2280,7 +2280,11 @@ if(BUILD_wireshark AND QT_FOUND)
 		${MINIZIP_LIBRARIES}
 	)
 
-	add_executable(wireshark WIN32 MACOSX_BUNDLE ${wireshark_FILES} ${EXTRA_BUNDLE_FILES})
+	if(ENABLE_APPLICATION_BUNDLE)
+		add_executable(wireshark WIN32 MACOSX_BUNDLE ${wireshark_FILES} ${EXTRA_BUNDLE_FILES})
+	else()
+		add_executable(wireshark WIN32 ${wireshark_FILES} ${EXTRA_BUNDLE_FILES})
+	endif()
 	if(WIN32)
 		set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT wireshark)
 	endif()
@@ -3196,11 +3200,11 @@ endif()
 
 install(
 	FILES
-		${CMAKE_MODULE_PATH}/FindGLIB2.cmake
-		${CMAKE_MODULE_PATH}/FindWSWinLibs.cmake
-		${CMAKE_MODULE_PATH}/UseAsn2Wrs.cmake
-		${CMAKE_MODULE_PATH}/LocatePythonModule.cmake
-		${CMAKE_MODULE_PATH}/UseMakePluginReg.cmake
+		cmake/modules/FindGLIB2.cmake
+		cmake/modules/FindWSWinLibs.cmake
+		cmake/modules/UseAsn2Wrs.cmake
+		cmake/modules/LocatePythonModule.cmake
+		cmake/modules/UseMakePluginReg.cmake
 	DESTINATION
 		${WIRESHARK_INSTALL_CMAKEDIR}
 )
